src/puppetlabs/trapperkeeper/authorization/acl.clj

```diff
@@ -51,12 +54,14 @@
 (def ACEChallenge
   "Pertinent authorization information extracted from a request used during
   authz rule enforcement."
-  {:certname schema/Str
-   :extensions Extensions})
+  {(schema/optional-key :certname) schema/Str
```

Comment:
This change here to the `ACEChallenge` schema lets the `allowed` function use an ACE that has the rbac permission string. This didn't seem like a great solution, but it was the only I could conjure up for the time being...

Reply:
This represents the information from the request that is used to determine whether the rule works, so if anything gets added wouldn't it be the token?